heft-plugins/heft-lint-plugin/src/LintPlugin.ts

```diff
@@ -126,20 +132,22 @@ export default class LintPlugin implements IHeftTaskPlugin<ILintPluginOptions> {
       // If we are not in the typescript phase, we need to create a typescript program
       // from the tsconfig file
       if (!inTypescriptPhase) {
+        const tsconfigPath: string = path.resolve(heftConfiguration.buildFolderPath, 'tsconfig.json');
         const tsProgram: IExtendedProgram = await this._createTypescriptProgramAsync(
           heftConfiguration,
           taskSession
         );
-        typescriptChangedFiles.push([tsProgram, new Set(tsProgram.getSourceFiles())]);
+        typescriptChangedFiles.push([tsProgram, new Set(tsProgram.getSourceFiles()), tsconfigPath]);
```

Comment:
Since the path is extractable from the program, you don't need to forward it separately.